### PR TITLE
Make sure error token set the new_line flag when necessary

### PR DESCRIPTION
**Pull request by Carreau**
Should solve #855

As far as I can tell, this should be a step toward the right thing to do. I'm going to assume it's likely not totally right and will make other test fail.

### CODE REVIEW

**Comment by Carreau** [comment]
Well, strangely test appear to to be happy, and it has been working locally. 
I'm still going to attempt writing a few more tests.

File: jedi/parser/tokenize.py [hunk 1]
```diff
@@ -233,6 +233,8 @@ def generate_tokens(readline, use_exact_op_types=False):
         if not line:
             if contstr:
                 yield TokenInfo(ERRORTOKEN, contstr, contstr_start, prefix)
+                if contstr.endswith('\n'):
+                    new_line = True
```
Comment (davidhalter) [inline jedi/parser/tokenize.py]: I like! Can you add a test for this one?

**Comment by Carreau** [comment]
> I like! Can you add a test for this one

Yes, thanks for reminding me. 
Done.

**Comment by davidhalter** [comment]
Thanks man and sorry for taking such a long time for a patch that's only a few LOC. :)

**Comment by Carreau** [comment]
it's ok, no hurry. I have plenty other things on my plate as well, so I can keep myself busy :-)

**Pull request merged**